zerver/lib/socket.py

```diff
@@ -152,6 +163,7 @@ def authenticate_client(self, msg):
         ioloop.remove_timeout(self.timeout_handle)
 
     def on_message(self, msg):
+        # type: (Dict[str, Any]) -> None
         log_data = dict(extra='[transport=%s' % (self.session.transport_name,))
         record_request_start_data(log_data)
         msg = ujson.loads(msg)
```

Comment:
So I think if we make the argument to `on_message` called `msg_raw`, that would have type str (so no need to use a Any)